packages/terra-date-input/tests/wdio/date-input-field-spec.js

```diff
@@ -1,30 +1,50 @@
 Terra.describeViewports('Date Input', ['medium'], () => {
-  describe('Date Input Field no date provide', () => {
+  // Temporary disabled test, since placeholder causes contrast accessibility failure.
```

Comment:
Do we have a follow-up issue to fix this?

Reply:
@nramamurth [Placeholders generally are problematic](https://developer.paciellogroup.com/blog/2011/02/html5-accessibility-chops-the-placeholder-attribute/) when trying to pass color contrast, as needing to pass the minimum 4.5:1 ratio if provided to properly meet the WCAG requirements, but doing so and setting a color that dark results in sighted users (normal acuity and low-vision alike) mistaking the placeholder text as an actual value. So for now - Terra and UX have been generally following older internal (not WCAG) color rules we used in the past (3:1 for placeholders). 

Going forward, UX is figuring out how to transition away from using placeholders, opting instead for better labels and help text, and instead leave empty form elements actually empty. There are quite a few usability articles that suggest this is a better way to go: [Nielsen Norman Group](https://www.nngroup.com/articles/form-design-placeholders/) and [UXmatters](https://www.uxmatters.com/mt/archives/2010/03/dont-put-hints-inside-text-boxes-in-web-forms.php), UX just needs to decide when to make the change and how to do so as the new Cerner standard.

Reply:
Added this issue to track the future work: https://github.com/cerner/terra-core/issues/2712